packages/sdk-core/src/modules/breadcrumbs/storage/InMemoryBreadcrumbsStorage.ts

```diff
@@ -53,6 +60,33 @@ export class InMemoryBreadcrumbsStorage implements BreadcrumbsStorage, Backtrace
 
         this._breadcrumbs.add(breadcrumb);
 
+        if (this._limits.maximumBreadcrumbsSize) {
```

Comment:
we're applying this rule twice. In the `BreadcrumbsManager` L:194 and here. Maybe we should always expect the storage to check it. I think it might make more sense if we assume the breadcrumb that exceed the limit will be dropped.

Reply:
it's not applied twice, in `BreadcrumbManager` `maximumBreadcrumbSize` is checked, and here `maximumBreadcrumbsSize`. I'll rename the latter to match `BreadcrumbLimit.maximumTotalBreadcrumbsSize`

Reply:
later you mean in another pull request?

Reply:
_latter_, not _later_.